funnel/views/email.py

```diff
@@ -1,15 +1,20 @@
+from collections import namedtuple
 from email.utils import formataddr, getaddresses
 
 from flask import current_app, render_template, request, url_for
 from flask_mailman import EmailMultiAlternatives
 
+from flask_babelhg import force_locale
 from html2text import html2text
 from premailer import transform
 
-from baseframe import _
+from baseframe import _, get_locale
```

Comment:
I was looking through Flask-Babelhg's code. Try importing this from `flask_babelhg`. It has a bit more of a safety harness.